### DISCUSSION TITLE
How to retrieve deleted repositories

### DISCUSSION THREAD

**Discussion by JulianHSC** [community/community#143426] [Repositories]
### Select Topic Area

Question

### Body

My repository name is "Listview-Pictures". I have deleted quite a number of them because I could not update them properly. Now my existing project is unstable and I need the deleted repositories back. Can someone help get them?

**Comment by davevad93** [comment]
Hi @JulianHSC ,

If your repo was deleted within 90 days, you can still recover it by following these steps:

1. In the upper-right corner  click your profile photo, then click `Settings`.

2. In the `Code planning, and automation` section of the sidebar, click `Repositories`.

3. Under `Repositories`, click `Deleted repositories`.

4. Next to the repository you want to restore, click `Restore`.

5. Read the warning, then click `I understand, restore this repository`.

**Comment by JulianHSC** [comment]
Thank you. I found the menus.
